debian/copyright

```diff
@@ -12,7 +12,6 @@ License: LGPL-3.0+
 
 Files: debian/*
 Copyright: 2016 Endless Computers INC
-Copyright: 2016 Niv Sardi <xaiki@endlessm.com>
```

Comment:
Your copyright is assigned to Endless as per your contract -- you have no sole ownership of this code :)